src/components/simulation/Additional_Resources.tsx

```diff
@@ -64,7 +64,7 @@ const Additional_Resources: React.FC<{ setStage: setStageType }> = ({
     e.preventDefault();
     window
       .open(
-        'https://www.capitalone.com/learn-grow/business-resources/',
+        'https://www.capitalone.com/learn-grow/money-management/',
```

Comment:
I updated this link from the business resource page to the money management page - which felt more relevant to this activity. Let me know if ya'll agree with this change and we can also discuss with the client.

Reply:
I think this is much better. Thanks!